scripts/create_dicts.py

```diff
@@ -880,6 +855,85 @@ def create_dicts(project):
         dict_object.post_process()
         dict_object.publish()
 
+    variable_module_location = Path(__file__).parent.parent / "process/data_structure"
+    module_names = [
+        file.name for file in variable_module_location.iterdir() if file.is_file()
+    ]
+
+    for module_name in module_names:
+        if module_name == "__init__.py":
+            continue
+        module = import_module(f"process.data_structure.{module_name.split('.', 1)[0]}")
+
+        module_tree = ast.parse(inspect.getsource(module))
+        initial_values_dict = {}
+        variable_names = []
+        var_names_and_descriptions = {}
+        dict_module_entry = {}
+        variable_types = {}
+
+        # get the variable names and initial values
+        for node in ast.walk(module_tree):
+            if isinstance(node, ast.AnnAssign):
+                # for each variable in the file, get the initial value
+                # (either is None, or value initialised in init_variables fn)
+                # set default to be None if variable is not being initialised eg if you
+                # just have `example_double: float` instead of `example_double: float = None`
+                initial_value = getattr(module, node.target.id, None)
```

Comment:
```suggestion
                initial_value = getattr(module, node.target.id)
```

If we remove this default then it will error if it cannot find the variable. I would suggest this is the desired behaviour because if it cannot find the variable here, it also cannot find it in the PROCESS models.